### ISSUE TITLE
Short(er) performance tests (to run as part of every build's sanity)

### ISSUE THREAD

**Issue by mykaul** [scylladb/scylla-cluster-tests#8840]
Summarizing a corridor talk I've had with @fruch and @yaronkaikov  :
The idea is to ensure a new AMI (With a new Linux kernel, or some backports) doesn't introduce a very obvious performance regression.
We should run a minimal (small) performance tests on a fixed (small) setup and see that we did not regress considerably.
Example:
- Fill cluster to 0.5TB (measure throughput)
- Read (measure latency)
- Bonus points : run repair, major compaction.

- [x] add a new job for this daily(per build) run (not the driver one) 
- [x] add a trigger job to SCT that would be triggered by scylla-pkg (similar to tier1)

**Comment by fruch** [comment]
@roydahan FYI

**Comment by fruch** [comment]
> The current tests aren't that long. If it's not for a local HW environment I don't see a reason to have a different test than the weekly one.

@roydahan  anything changed in that regards ? can we move this into the backlog ?

**Comment by roydahan** [comment]
Yes, I want to use the shorter test that @juliayakovlev did for the java driver, nightly (triggered by new AMI).
However, we must nail down the suspected flakiness first.

Let's have it on @juliayakovlev queue with high priority and decide next week what we can already trigger.

**Comment by juliayakovlev** [comment]
@fruch @roydahan 
1. The java driver test is defined to be run on SCT master. Do we want to continue with that or create a new one in branch v16?
2. What about operations that Yaniv asked (repair, major compactions)? Do we want to trigger that?

**Comment by roydahan** [comment]
1. should be switched to perf-v16.
2. Let's start first at least with what we have, but let's track the costs as well.

**Comment by roydahan** [comment]
We need to do it through a dashboard we have once it runs for a while (week or two).

**Comment by juliayakovlev** [comment]
> 1. should be switched to perf-v16.

Created backport PR https://github.com/scylladb/scylla-cluster-tests/pull/10660

> 3. Let's start first at least with what we have, but let's track the costs as well.

**Comment by juliayakovlev** [comment]
Call sanity  trigger job with every ami:
https://github.com/scylladb/scylla-pkg/pull/5070

**Comment by juliayakovlev** [comment]
https://github.com/scylladb/scylla-pkg/pull/5090
Another solution for run same performance sanity job for master and release

**Comment by roydahan** [comment]
Reopening because I want to review the costs and results and make sure we get something out of it.
@juliayakovlev please schedule a meeting with me to review this.

**Comment by juliayakovlev** [comment]
According to meeting with @roydahan and @fruch :

1. checked the sanity test monthly cost and created the cost report
2. the test results are not stable, we need to continue to looking after the results

**Comment by roydahan** [comment]
Not sure if moving to Done is the best thing to do here.
We need to investigate the flakiness and take decisions if it worth running it like this.
If not, possibly running it on dedicated host / lab server / something else.

**Comment by fruch** [comment]
> Not sure if moving to Done is the best thing to do here. We need to investigate the flakiness and take decisions if it worth running it like this. If not, possibly running it on dedicated host / lab server / something else.

task of having a shorter test running on every build is done.

what you are referring to is not this task anymore, it's something else... and won't be tracked in this item...

**Comment by roydahan** [comment]
I don't mind closing it as long as we have a follow up task for the rest of the work.
I don't want it to be neglected and just run there without giving us any value.

BTW, it would also be good to summarize here the monthly estimated cost that we've seen yesterday for this daily addition.

**Comment by fruch** [comment]
> I don't mind closing it as long as we have a follow up task for the rest of the work. I don't want it to be neglected and just run there without giving us any value.
> 
> BTW, it would also be good to summarize here the monthly estimated cost that we've seen yesterday for this daily addition.

we'll take it to qa-tasks, not sure I want to talk $$ here :)
